indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -47,4 +48,154 @@
+            headers: {
+                0: {
+                    sorter: false
+                }
+            }
+        });
+    }
+
+    function setupRegistrationsList() {
+        colorizeSelectedRows();
+        setupStaticURLGeneration();
+        setupTableSorter();
+
+        $('.registrations .toolbar')
+        .dropdown()
+        .on('click', '.js-dialog-action', function(e) {
+            e.preventDefault();
+            var $this = $(this);
+            ajaxDialog({
+                dialogClasses: 'reglist-filter-dialog',
+                trigger: this,
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        $('.registrations-table-wrapper').html(data.registrations_list);
+                        colorizeSelectedRows();
+                        setupTableSorter();
+                    }
+                }
+            });
+        });
+
+        $('#select-all').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', true).trigger('change');
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+    }
+
+    function colorizeSelectedRows() {
+        $('table.i-table input.select-row').on('change', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+        });
+    }
+
+    function colorizeFilter(filter) {
+        filter.toggleClass('active', filter.find(':checked').length > 0);
+    }
+
+    function colorizeActiveFilters() {
+        $('.reglist-filter .filter').each(function() {
+            colorizeFilter($(this));
+        });
+    }
+
+    global.setupRegistrationsListFilter = function setupRegistrationsListFilter() {
+        $('.reglist-filter').dropdown({selector: '.reglist-column .title'});
+        colorizeActiveFilters();
+
+        var visibleColumnsRegItemsField = $('#visible-columns-reg-items');
+        var regItemsData = JSON.parse(visibleColumnsRegItemsField.val());
+
+        $('.reglist-column')
+        .on('click', '.trigger', function() {
+            var $this = $(this);
+            var field = $this.closest('.reglist-column');
+            var fieldId = field.data('id');
+            var data = regItemsData;
+            var enabled = $this.hasClass('enabled');
+
+            if (enabled) {
+                data.splice(data.indexOf(fieldId), 1);
+            }
+            else if (!enabled) {
+                data.push(fieldId);
+            }
+            $this.toggleClass('enabled', !enabled);
+            field.toggleClass('striped', enabled);
+            visibleColumnsRegItemsField.val(JSON.stringify(data)).trigger('change');
+        })
+        .each(function() {
+            var field = $(this);
+            var fieldId = field.data('id');
+            var data = regItemsData;
```

Comment:
it's true, now that we got rid of the user_info, we don't need the `var data` anymore